tango/integrations/beaker/executor.py

```diff
@@ -436,6 +440,17 @@ def __init__(
                 "or as the environment variable 'GITHUB_TOKEN'."
             )
 
+        self.google_token = google_token or os.environ.get("GOOGLE_TOKEN")
+        # If the user passes file path instead of actual credentials dict, load the credentials
+        # so that they can be used in beaker.
```

Comment:
This would be the scenario if someone used gcloud auth locally, and then runs stuff on beaker. We can insist that people manually copy the contents of the file into a string, and add that as an environment variable. Just thought that this would be convenient. Passing the filepath as token does work locally with gcsfs, so this extends that to `BeakerExecutor`.